web/models/user.ex

```diff
@@ -32,6 +39,14 @@ defmodule CodeCorps.User do
     |> put_pass_hash()
   end
 
+  def update_changeset(struct, params) do
+    struct
+    |> changeset(params)
+    |> cast(params, [:first_name, :last_name, :twitter, :biography, :website])
+    |> validate_format(:website, ~r/\A((http|https):\/\/)?[a-z0-9]+([\-\.]{1}[a-z0-9]+)*\.[a-z]{2,}(([0-9]{1,5})?\/.*)?#=\z/ix)
+    |> validate_format(:twitter, ~r/\A[a-zA-Z0-9_]{1,15}\z/)
+  end
```

Comment:
What does `validate_format` do when the values are blank? In other words, if I try to update my profile but never added a website or Twitter, will it fail?